emails/utils.py

```diff
@@ -132,29 +132,30 @@ def ses_send_raw_email(
     return HttpResponse("Sent email to final recipient.", status=200)
 
 
-def ses_relay_email(
-        from_address, relay_address, subject, message_body, attachments
-):
+def ses_relay_email(user_profile, from_address, subject,
+                    message_body, attachments, address):
```

Comment:
Oh, hmm ... what's the advantage of passing the `user.email` value instead of passing in the `user_profile` object? I typically find that passing objects into functions helps make the function logic more independent from the calling code. So this function is more like "just give me a profile and I'll figure out the email address".